apps/bare-expo/ios/Podfile.lock

```diff
@@ -562,7 +562,7 @@ PODS:
     - React-Core
   - react-native-slider (4.2.4):
     - React-Core
-  - react-native-view-shot (3.4.0):
+  - react-native-view-shot (3.5.0):
```

Comment:
@douglowder shouldn't we update `react-native-netinfo (9.3.7)` here instead?

Reply:
Hmm yes, I will correct that in the next PR